### DISCUSSION TITLE
Installation confirmation for every installer is annoying

### DISCUSSION THREAD

**Discussion by FraH90** [microsoft/winget-cli#576] [General]
So, I've started using Winget today. I've seen that every package that I try to install with winget it doesn't do it silently, but it asks every time if I wanna execute the installer.

Shouldn't be automatic installation the purpose of Winget? I put a list of packages that I need to install on my machine and it will do it automatically? Without user interaction?

So I'd like to know if I'm missing something, if there is a command options that I need to provide to Winget to get a completely silent installation or this is the default current behaviour, and maybe it will be changed in the future when winget will get more mature

**Comment by megamorf** [comment]
Run cmd/PowerShell as administrator. This provides winget and its child processes (the installers) with an administrator access token so that UAC will not request manual elevation.

PS: This is not winget specific. That's just how Windows works.
